src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -562,6 +571,15 @@ public boolean isOnGround() {
     public void remove() {
         callEvent(PlayerDisconnectEvent.class, new PlayerDisconnectEvent(this));
 
+        // todo issue not caused by any of this
+        // tab list
```

Comment:
Could you clarify what is meant by this comment?

Reply:
We had an issue due to players logging out and having all the players removed from the TabList, it has since been fixed. Comment will be removed :D